src/components/questions/Questions.jsx

```diff
@@ -13,6 +14,7 @@ const Questions = () => {
       <SearchBar/>
       <QuestionList data={data}/>
       <AddQuestion/>
+      <QuestionForm/>
```

Comment:
I'm having a little bit of a hard time understanding this. It looks like there is also a `<QuestionForm/>` inside of `<AddQuestion/>`. Are both of them necessary?